### PR TITLE
Improvement to python-telegram-bot

### PR DESCRIPTION
**Pull request by annihilatorrrr**
* Better code and start/stop.

* Update Procfile to worker as using long polling method.

* Update README.md to correct some info.

### CODE REVIEW

File: examples/python-telegram-bot/Procfile [hunk 1]
```diff
@@ -1 +1 @@
-web: python bot.py
\ No newline at end of file
+worker: python3 bot.py
```
Comment (Milo123459) [inline examples/python-telegram-bot/Procfile:1]: This is a bad change. It has to be `web`.

File: examples/python-telegram-bot/README.md [hunk 2]
```diff
@@ -20,9 +20,9 @@ This example starts a Telegram bot using [python-telegram-bot](https://www.pytho
 
 ## 💁‍♀️ How to use
 
-- Install packages using `pip install -r requirements.txt`
-- Start the bot using `python bot.py`
+- Install packages using `pip3 install -r requirements.txt`
```
Comment (Milo123459) [inline examples/python-telegram-bot/README.md:23]: These 2 changes really aren't needed. It runs with Python 3 anyway.

File: examples/python-telegram-bot/Procfile (hunk 1 above)
Comment (annihilatorrrr) [reply examples/python-telegram-bot/Procfile:1]: Oo i see

File: examples/python-telegram-bot/README.md (hunk 2 above)
Comment (annihilatorrrr) [reply examples/python-telegram-bot/README.md:23]: 😅

**Pull request merged**